views/index.html

```diff
@@ -306,6 +306,20 @@
 {{/unless}}
 
 <script src="{{static}}/js/vendor/jquery-1.11.0.min.js"></script>
+<script>
+  // Load libraries from a server config file
+  // This method is used so we can both expand vars, and support <script> in the json strings
+  window.libraries = window.libraries || [];
+  $.ajax({
+    url: '{{root}}/{{config.libs.jsLibPath}}',
```

Comment:
This is wrong, we don't serve static content over the default root, it should use the static path.